hackathon_site/dashboard/frontend/src/components/cart/CartSummary/CartSummary.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { render, waitFor } from "@testing-library/react";
+import CartSummary from "components/cart/CartSummary/CartSummary";
+import { mockCartItems } from "testing/mockData";
+import store, { RootState } from "slices/store";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import thunk, { ThunkDispatch } from "redux-thunk";
+import { initialState, cartReducerName } from "slices/hardware/cartSlice";
+import { post } from "api/api";
+import { fireEvent, makeMockApiListResponse, when } from "testing/utils";
+import { AnyAction } from "redux";
+
+jest.mock("api/api", () => ({
+    ...jest.requireActual("api/api"),
+    post: jest.fn(),
+}));
+
+const mockedPost = post as jest.MockedFunction<typeof post>;
+const orderUri = "/api/hardware/orders/";
+
+const mockState: RootState = {
+    ...store.getState(),
+    [cartReducerName]: initialState,
+};
+
+type DispatchExts = ThunkDispatch<RootState, void, AnyAction>;
+const mockStore = configureStore<RootState, DispatchExts>([thunk]);
+
+describe("Render cartQuantity", () => {
+    test("Checks for correct POST request", async () => {
+        const response = makeMockApiListResponse(mockCartItems);
+        when(mockedPost).calledWith(orderUri, {}).mockResolvedValue(response);
+
+        const store = mockStore(mockState);
+        const { getByTestId } = render(
+            <Provider store={store}>
+                <CartSummary />
+            </Provider>
+        );
+
+        const submitOrderBtn = getByTestId("submit-order-button");
+
+        fireEvent.click(submitOrderBtn);
+
+        await waitFor(() => {
+            expect(getByTestId("cart-quantity-total")).toHaveTextContent("0");
```

Comment:
Nice job on this test, 
This test may be better to put in Cart.test.tsx because it's a big part of the Cart page but can you also make an assertion that the API was called, something like 
`expect(mockedPost).toHaveBeenCalledWith(orderUri)`
and after you move it to Cart.test.tsx, you can assert that all items on the cart page have disappeared.

Also, was there a unit test here before? I remember there was one just testing the total cart items were displaying properly, probs should bring that test back.